src/app/parcels/ActionBar/Statuses.tsx

```diff
@@ -89,6 +71,25 @@ const Statuses: React.FC<Props> = ({
     const [selectedParcels, setSelectedParcels] = useState<ParcelsTableRow[]>([]);
     const [selectedStatus, setSelectedStatus] = useState<statusType | null>(null);
     const [statusModal, setStatusModal] = useState(false);
+    const [parcelStatuses, setParcelStatuses] = useState<ParcelStatus[] | null>(null);
+
+    useEffect(() => {
+        const getParcelStatuses = async (): Promise<void> => {
+            const { data: parcelStatusesData, error: parcelStatusesError } =
+                await fetchParcelStatuses();
+            if (parcelStatusesError) {
+                switch (parcelStatusesError.type) {
+                    case "failedToFetchStatuses":
+                        setModalError(
+                            `Unable to retrieve statuses filter for parcels. Log ID: ${parcelStatusesError.logId}`
```

Comment:
I'm not sure where to display the error but modal error appears next to the status dropdown

Reply:
Yeah I think that's fine for now